framework/test_app/runners/k8s/gamma_server_runner.py

```diff
@@ -103,13 +100,14 @@ def __init__(
             namespace_template=namespace_template,
             debug_use_port_forwarding=debug_use_port_forwarding,
             enable_workload_identity=enable_workload_identity,
+            enable_csm_observability=enable_csm_observability,
```

Comment:
instead, let's put all this into `deployment_args`, see 
https://github.com/grpc/psm-interop/blob/2eea2ff909d58f95917c2f5515bdb705704fc22d/framework/test_app/runners/k8s/k8s_xds_server_runner.py#L34-L36